core/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -41,6 +41,11 @@
  * @param <T> The concrete type of this builder.
  */
 public abstract class ServerBuilder<T extends ServerBuilder<T>> {
+
+  public static ServerBuilder<? extends ServerBuilder<?>> forAddress(String name, int port) {
```

Comment:
After thinking about it, it feels weird that it doesn't match the Client Provider interface.  How would you feel about just throwing here if the name parameter is not empty string for now, and keeping the API the same.  When we are ready to expose the name, we can remove the check?